#! /bin/sh /usr/share/dpatch/dpatch-run
## 03_gcc43_include.dpatch by Michael Meskes <meskes@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Add missing include directives for gcc-4.3

@DPATCH@

diff -ruN tora-1.3.22.orig/ext/loki/loki-0.1.6/src/SmallObj.cpp tora-1.3.22/ext/loki/loki-0.1.6/src/SmallObj.cpp
--- tora-1.3.22.orig/ext/loki/loki-0.1.6/src/SmallObj.cpp	2006-10-17 22:05:42.000000000 +0200
+++ tora-1.3.22/ext/loki/loki-0.1.6/src/SmallObj.cpp	2008-02-11 11:04:08.000000000 +0100
@@ -21,6 +21,7 @@
 #include <cassert>
 #include <vector>
 #include <bitset>
+#include <limits.h>
 
 //#define DO_EXTRA_LOKI_TESTS
 //#define USE_NEW_TO_ALLOCATE
